src/test/resources/jacocoTest/build.sbt

```diff
@@ -0,0 +1,19 @@
+import de.johoop.jacoco4sbt.Thresholds
+
+name := "jacocoTest"
+organization := "com.navetas"
+
+scalaVersion := "2.10.4"
+scalacOptions ++= Seq("-deprecation", "-optimize", "-unchecked", "-Xlint", "-language:_")
+
+libraryDependencies += "org.scalatest" %% "scalatest" % "3.0.4" % "test"
+
+jacoco.settings
+fork in jacoco.Config := false
+jacoco.thresholds in jacoco.Config := Thresholds(
```

Comment:
1. The key names should be prefixed as `jacocoThresholds` - http://www.scala-sbt.org/1.0/docs/Plugins-Best-Practices.html#Avoid+namespace+clashes
2. Consider task scoping instead of custom config.

Reply:
Same comment as above.